clients/src/main/java/org/apache/kafka/clients/consumer/MockConsumer.java

```diff
@@ -70,7 +70,7 @@
     private boolean shouldRebalance;
 
     public MockConsumer(OffsetResetStrategy offsetResetStrategy) {
-        this.subscriptions = new SubscriptionState(new LogContext(), offsetResetStrategy);
+        this.subscriptions = new SubscriptionState(new LogContext(), offsetResetStrategy, 100, 100);
```

Comment:
If we pass the same value for `retryBackoffMs` and `retryBackoffMaxMs`, a static backoff of 100 ms will be applied. I was thinking that we'd better make the MockConsumer use the static backoff since some of the written tests are assuming that.